### ISSUE TITLE
Invalid module name in augmentation. Module '@sparkfabrik/react-native-idfa-aaid' resolves to an untyped module 

### ISSUE THREAD

**Issue by iremcalmazz** [sparkfabrik/sparkfabrik-react-native-idfa-aaid#57]
I installed the module @sparkfabrik/react-native-idfa-aaid as "npm install @sparkfabrik/react-native-idfa-aaid"

It works fine with .js files but when i try to use this module on .tsx file, it gives me an error. 

Error on VSCode : Invalid module name in augmentation. Module '@sparkfabrik/react-native-idfa-aaid' resolves to an untyped module at 'c:/Users/..../.../node_modules/@sparkfabrik/react-native-idfa-aaid/lib/commonjs/index.js', which cannot be augmented.

Is there any  tTypeScript definitions for the library ?

**Comment by edodusi** [comment]
Hi @iremcalmazz, I fixed the types path and published a new release `1.0.1`, can you please try it?

**Comment by sekoyo** [comment]
Hm mine started randomly breaking on 1.0.1 and 1.1.1 in VSCode and `tsc`, strange, I can see package.json has a typings entry. All other NPM modules are fine.

```
Could not find a declaration file for module '@sparkfabrik/react-native-idfa-aaid'. '......../node_modules/@sparkfabrik/react-native-idfa-aaid/lib/commonjs/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/sparkfabrik__react-native-idfa-aaid` if it exists or add a new declaration (.d.ts) file containing `declare module '@sparkfabrik/react-native-idfa-aaid';`ts(7016)
```
